airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/source/relationaldb/state/SourceStateIterator.kt

```diff
@@ -24,7 +24,7 @@ open class SourceStateIterator<T>(
     private var lastCheckpoint: Instant = Instant.now()
 
     override fun computeNext(): AirbyteMessage? {
-        var iteratorHasNextValue = false
+        var iteratorHasNextValue: Boolean
```

Comment:
fyi you can also `var iteratorHasNextValue: Boolean = try { ...` though there's no benefit to readability whatsoever in this case